tasks/main.yml

```diff
@@ -38,10 +38,14 @@
   notify: Restart Caddy
 
 - name: Create directories
-  file: path={{ item }} state=directory owner={{ caddy_user }} mode=0700
+  file: path={{ item.dir }} state=directory owner={{ item.user|default(caddy_user) }} group={{ item.group|default('root') }} mode={{ item.mode|default(750) }}
```

Comment:
True, it is more complicated. But most of the time though I see log directories in /var/log owned by the daemon user, and group root.

But as root has access already, it shouldn't matter much. Could be reverted if you wish :)